pkg/registry/core/trace/registry_server.go

```diff
@@ -96,7 +98,8 @@ func (t *traceRegistryServer) BulkRegisterNSE(server registry.NetworkServiceRegi
 
 func (t *traceRegistryServer) RemoveNSE(ctx context.Context, request *registry.RemoveNSERequest) (*empty.Empty, error) {
 	// Create a new span
-	span := spanhelper.FromContext(ctx, fmt.Sprintf("%s.Request", typeutils.GetTypeName(t.traced)))
+	operation := fmt.Sprintf("%s/%s.RegisterNSE", typeutils.GetPkgPath(t.traced), typeutils.GetTypeName(t.traced))
```

Comment:
```suggestion
	operation := fmt.Sprintf("%s/%s. RemoveNSE", typeutils.GetPkgPath(t.traced), typeutils.GetTypeName(t.traced))
```

Reply:
Fixed in: https://github.com/networkservicemesh/sdk/pull/183